.github/workflows/build.yml

```diff
@@ -528,7 +528,7 @@ jobs:
         run: yarn lerna run test --scope @sentry/profiling-node
 
   job_browser_playwright_tests:
-    name: Playwright (${{ matrix.bundle }}${{ matrix.shard && format(' {0}/{1}', matrix.shard, matrix.shards) || ''}}) Tests
+    name: PW ${{ matrix.bundle }}${{ matrix.project && matrix.project != 'chromium' && format(' {0}', matrix.project) || ''}}${{ matrix.shard && format(' ({0}/{1})', matrix.shard, matrix.shards) || ''}} Tests
```

Comment:
I'd really prefer if we kept `Playwright` as `PW` is not obvious for an onlooker.

Reply:
The reason for this is that as of now this is so long that you often miss a lot of context: 
![image](https://github.com/user-attachments/assets/504413e5-5db1-4794-bab3-ea184dabd09f)

making it very hard to find what kind of test this actually is :(

Reply:
I named it back to `Playwright ...`!